packages/backend/src/peripherals/database/AssetRepository.ts

```diff
@@ -70,8 +72,8 @@ export class AssetRepository extends BaseRepository {
     return row ? toAssetDetailsRecord(row) : undefined
   }
 
-  async getDetailsByAssetHashes(
-    assetHashes: AssetHash[]
+  async getDetailsByAssetHashesOrTypeHashes(
+    assetHashes: (AssetHash | Hash256)[]
```

Comment:
Asset type is not `Hash256`. It's also `AssetHash`. I know it's confusing, but they share the type.